client/blocks/get-apps/desktop-download-card.jsx

```diff
@@ -63,8 +63,8 @@ class DesktopDownloadCard extends Component {
 		}
 	}
 
-	getPlatformImage( platform ) {
-		switch ( platform ) {
+	getPlatformImage( platformOrLink ) {
```

Comment:
I am trying to understand how `platformOrLink` is declared or where it is coming from. Maybe you could give me a bit more information?

Reply:
`platformOrLink` is the name of the parameter, in the line 66 we are declaring a method (a.k.a function inside a class) that accepts one parameter called `platformOrLink`.
I preferred to rename it because it will be more clear for folks to use it.